events/README.md

```diff
@@ -0,0 +1,416 @@
+  'timestamp': {
+    'createdAt': new Date(),
+    'updatedAt': new Date(),
+  }
+}
+```
+
+## Requests
+
+| Route                                                       | Description                                                            |
+| ----------------------------------------------------------- | ---------------------------------------------------------------------- |
+| [POST /rooms](#post---rooms)                                | Create a new room, either randomly or with the requested configuration |
+| [GET /rooms](#get-rooms)                                    | Get all the rooms                                                      |
+| [POST /join](#post---join)                                  | Generate an auth token for a peer to join a room                       |
+| [GET /rooms/:id=<ROOM_ID>](#get---roomsidroom_id)           | Retrieves the details of a specific active room.                       |
+| [PUT /rooms](#put---rooms)                                  | Update the room, make room enabled/disabled,                           |
+| [DELETE /rooms](#delete---rooms)                            | Trigger this request to end an active room.                            |
+| [GET /session](#get---session)                              | To retrieve all the sessions.                                          |
+| [GET /session/:status](#get---sessionstatus)                | To get currently running session.                                      |
+| [GET /session/:id=<SESSION_ID>](#get---sessionidsession_id) | Retrieves the details of a specific active session.                    |
+
+## POST - /rooms
+
+Create a new room, either randomly or with the requested configuration.
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - Optional: `name=[string]` (`name` is the name of the room which is case-insensitive. Accepted characters are `a-z, A-Z, 0-9, and . - : _`)
+  - Optional: `description=[string]` (`description` describes the usage of the room.)
+  - Optional: template_id=[string] (template_id which can be found from dashboard)
+  - Optional: region=[string] (region in which you want to create room.)
+  ```json
+  {
+    "name": "new-room-1662723668",
+    "description": "This is a sample description for the room",
+    "template_id": "<template_id that you wish to associate w/ the room>",
+    "region": "in"
+  }
+  ```
+- **Headers**
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "id": "<id>",
+        "name": "<room_name>",
+        "enabled": true,
+        "description": "This is a sample description for the room",
+        "customer": "<customer_id>",
+        "recording_info": {
+          "enabled": false
+        },
+        "template_id": "<template_id>",
+        "region": "in",
+        "created_at": "",
+        "updated_at": ""
```

Comment:
can we also have information about the expected format of the `created_at` and `updated_at` fields in the response body

Reply:
Yes, added it.